packages/fuels/tests/scripts.rs

```diff
@@ -192,40 +199,6 @@ async fn test_script_call_with_non_default_max_input() -> Result<()> {
     Ok(())
 }
 
-#[tokio::test]
```

Comment:
Why are we deleting this test?

Reply:
This tested that the script tx signature is ok. `utxo_validation` is turned on by default in tests,  which means this test doesn't test anything new. We have this script call several times in our tests.